src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -189,7 +189,11 @@ def apply(cls, func_node: ast.FunctionDef, context: dict):
     def __init__(self, context: dict):
         self.context = context
         self.current_block = None
-        self.all_skip_names = set(gtscript.builtins | {"gt4py", "gtscript"})
+        self.all_skip_names = set(
```

Comment:
For us it was not clear that these are actual gt4py builtins as it seems that these are what make up the gt4py language whereas the pythonic name of native function calls seem to be something different. This is why we opted for that approach. But since you feel like this is the better approach we changed it.